ui/src/plugins/dev.perfetto.RecordTraceV2/adb/adb_tracing_session.ts

```diff
@@ -18,15 +18,37 @@ import {TracingProtocol} from '../tracing_protocol/tracing_protocol';
 import {errResult, okResult, Result} from '../../../base/result';
 import {exists} from '../../../base/utils';
 import {ConsumerIpcTracingSession} from '../tracing_protocol/consumer_ipc_tracing_session';
+import {LOCAL_ABSTRACT_SOCKET_FLAG} from '../local_flag';
 
-export const CONSUMER_SOCKET = '/dev/socket/traced_consumer';
+function useCustomSocket(): boolean {
+  if (LOCAL_ABSTRACT_SOCKET_FLAG.flag === undefined) {
+    return false;
+  }
+  return LOCAL_ABSTRACT_SOCKET_FLAG.flag.get();
+}
+
+function getConsumerSocketType() {
```

Comment:
do we ever need only one of them?
If not, could these two be folded into a single

getConsumerSocketForAdb() which returns either localabstract:traced.... or localfs:/dev/socket/... ?

Reply:
There are two accessors now: 
- `getTracedConsumerSocketAddressForAdb` which returns the address formated to be used with ADB.
- `getTracedConsumerSocketAddress` which returns the address stored in the settings. For any information printed to the user, it should be better to use this name as it matches what has been set in the setting.